### DISCUSSION TITLE
Voicemail icon

### DISCUSSION THREAD

**Discussion by thomaschaaf** [twilio-labs/paste#2407] [Request: Icon] [answered]
I am building an interface in flex where workers can see their voicemails. It would be nice to have an icon for this.

**Comment by TheSisb** [comment]
Hi @thomaschaaf,

We have some work planned in our backlog to add these (the other threads + this one) icons. No ETA yet, but hopefully coming soon.

**Comment by SiTaggart** [comment, marked as answer]
Hi @thomaschaaf 

The Voicemail icon is now available to use in Figma an NPM https://paste.twilio.design/components/icons
